handbook/product/rollout_process.md

```diff
@@ -0,0 +1,47 @@
+# Rollout Process
+
+The rollout process consists of *Pre-Launch* phase and *Deployment* phase. We can push changes to Sourcegraph.com when we
+want as we own the deployment process. For enterprise instances of Sourcegraph.com, we push a new release monthly and our
+clients decide to upgrade to the latest release when they want.
+
+## Sourcegraph.com
+
+### Pre-Launch
+**User Testing** 
+- Hallway test with internal users.
+- Open the feature to internal users by turning it on for the Sourcegraph organization. Make sure it's opened with enough
+time left before planned launch date to receive and address all the feedback received.  Follow this [document](https://about.sourcegraph.com/handbook/engineering/distribution/update_sourcegraph_website) to enable your feature in site settings and turn it on only for internal users in the Sourcegraph organization. TODO: Update doc linked above to add a bit more content for new teammates
+
+**Design** 
+(skip if it doesn't apply)
+- Test the feature for both themes, Light and Dark. Check for icon consistency across themes.
+* Test on small, medium, large and extra-large screen sizes.
+  * Small screens are important to consider for a good experience when the window is resized or in side-by-side mode.
+* Code examples in design
+  * Font should be "code font" or monospace.
+  * Easy to copy and paste (no fancy quotes).
```

Comment:
I think these things belong more into a style guide than into a rollout process

Reply:
Agreed, all of this should have been caught in design review.

Reply:
Agree, I'd like to link to a style guide here eventually. I see this style guide living in `/design`.